power-platform/admin/environment-groups.md

```diff
@@ -58,7 +59,7 @@ Enabling _default environment routing_ might give you more environments to manag
 
 ## Configure the rules for your environment group
 
-After you create the environment group, Power Platform tenant administrators can immediately add managed environments or configure the group's rules. Currently, there are six available rules, which are the same capabilities found in [Managed Environment](managed-environment-overview.md). 
+After you create the environment group, Power Platform tenant administrators can immediately add managed environments or configure the group's rules. The available rules are the same capabilities found in [Managed Environment](managed-environment-overview.md). 
```

Comment:
this part is not true, cross-geo is not an ME capability

Reply:
@mikferland-msft should we just delete the second sentence here?

Reply:
We can delete the 2nd sentence "The available rules are the same capabilities found in [Managed Environment](managed-environment-overview.md)." Thanks!

Reply:
updated